src/services/juno/custom/helpers/admin-resource.ts

```diff
@@ -10,19 +10,30 @@ import { customApi } from "../custom";
 export const AP_ID = 0;
 export const REVIEWER_ID = 0.5;
 
-export function getCWs(id: number) {
-  //TODO: atm, only two admin types, refactor this once > 2
-  //charities doesn't have hardcoded cws, so only test for AP_ID && REVIEWER_ID
-  const cw3 = id === AP_ID ? contracts.cw3ApTeam : contracts.cw3ReviewTeam;
-  const cw4 =
-    id === AP_ID ? contracts.cw4GrpApTeam : contracts.cw4GrpReviewTeam;
-  const type: AdminResources["type"] = id === AP_ID ? "ap" : "review";
-  return { cw3, cw4, type };
-}
+type Admins = AdminResources["type"];
+const _charity: Admins = "charity";
+type CWs = {
+  [key: number]:
+    | {
+        cw3: string;
+        cw4: string;
+        type: Exclude<Admins, typeof _charity>;
+      }
+    | undefined;
+};
 
-export function isAp(id: number) {
-  return id === AP_ID || id === REVIEWER_ID;
-}
+export const apCWs: CWs = {
+  [AP_ID]: {
+    cw3: contracts.cw3ApTeam,
+    cw4: contracts.cw4GrpApTeam,
+    type: "ap",
+  },
+  [REVIEWER_ID]: {
+    cw3: contracts.cw3ApTeam,
+    cw4: contracts.cw4GrpApTeam,
+    type: "ap",
```

Comment:
```suggestion
    type: "review",
```

Reply:
Using AP params for `REVIEWER_ID` is a mistake in the example given in https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1856#discussion_r1115678467, should use Reviewer params.